build/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
```

Comment:
Do we have any instruction w.r.t how users should be using this Jenkinsfile?

Reply:
Yes, but it can be much clearer. I will add a note about the need of copying `Jenkinsfile` and `tf-wrapper.sh` to the root folders or the new repos (when people distribute the mono-repo to multi-repo)